docs(tutorials): add pull-over and pull-out scenario in tutorials page (#510)
autowarefoundation/autoware-documentation: docs/tutorials/ad-hoc-simulation/planning-simulation.md

   ![parking-maneuver](images/planning/parking/parking-maneuver.png)

> ### Pull-over and pull-out scenario

Reviewer: ```suggestion
### Pull over and pull out scenario
```

> ### Pull over and pull out scenario

Reviewer: ```suggestion
### Pull out and pull over scenario
```

> 1. Begin by positioning the ego-vehicle at the roadside.

Reviewer: ```suggestion
1. Begin by positioning the ego vehicle at the road shoulder.
```

> 1. Begin by positioning the ego vehicle at the road shoulder.

Reviewer: ```suggestion
1. In a pull out scenario, set the ego vehicle at the road shoulder.
```

   ![pullover-pullout](images/planning/pullover-pullout/pullover-pullout.jpg)

> 2. set a destination and then activate the vehicle.

Reviewer: ```suggestion
2. Set a destination and then engage the ego vehicle.
```

> 2. Set a destination and then engage the ego vehicle.

Reviewer: ```suggestion
2. Set a goal and then engage the ego vehicle.
```

> 3. In a pull-over scenario, similarly position the ego-vehicle in a lane and set a destination to the roadside.

Reviewer: ```suggestion
3. In a pull-over scenario, similarly set the ego vehicle in a lane and set a destination on the road shoulder.
```

> 3. In a pull-over scenario, similarly set the ego vehicle in a lane and set a destination on the road shoulder.

Reviewer: ```suggestion
3. In a pull over scenario, similarly set the ego vehicle in a lane and set a goal on the road shoulder.
```
